crypto/src/traits.rs

```diff
@@ -102,6 +106,7 @@ pub trait VerifyingKey:
 pub trait SigningKey: ToFromBytes + Serialize + DeserializeOwned + Send + Sync + 'static {
     type PubKey: VerifyingKey<PrivKey = Self>;
     type Sig: Authenticator<PrivKey = Self>;
+    const LENGTH: usize;
```

Comment:
What's the intent of this LENGTH? I understand it may afford you checks when manipulating an aggregate signature, but I would expect some uses of `Self::LENGTH` in the code to reveal the utility.

Reply:
It's pretty useful in the Sui repo.